### PR TITLE
Split the program name and desc for styling purposes

### PR DESCRIPTION
**Pull request by ivonne-hernandez**
What (if anything) did you refactor?
- I split the program name and description for styling purposes.

Were there any issues that arose?
- [ ] I noticed that an `ErrorMessage` component does not render when the user selects `Yes` but then doesn't check any of the boxes.

Is there anything that you need from your teammate?
- [ ] Please go through and make sure that nothing breaks. 
- [ ] I had to refactor the `Confirmation` component to account for the `category_benefits` schema. Please note that once this is approved, I'll have to go in and manually update all of these FM values.

Updated text and styling on `step-8`:
Screenshot 2024-08-08 at 4 12 59 PM

Updated `Confirmation` component:
Screenshot 2024-08-28 at 4 52 59 PM

### CODE REVIEW

File: src/Components/CheckboxGroup/BasicCheckboxGroup.js [hunk 1]
```diff
@@ -27,13 +27,25 @@ const BasicCheckboxGroup = ({ stateVariable, options }) => {
     const optionKeys = Object.keys(options);
 
     const formControlLabels = optionKeys.map((optionKey) => {
+      const splitLabel = options[optionKey].props.defaultMessage.split(':');
```
Comment (CalebPena) [inline src/Components/CheckboxGroup/BasicCheckboxGroup.js:30]: Does this work for Spanish? You might have to split them into 2 separate messages in the config.

**Review by CalebPena — CHANGES_REQUESTED** [review]

File: src/Components/Confirmation/Confirmation.js [hunk 2]
```diff
@@ -564,17 +564,33 @@ const Confirmation = () => {
...
-        <p key={option} className="bottom-margin">
-          {' '}
-          {relatedOptionsList[option]}{' '}
-        </p>
-      );
-    });
+  const listAllTruthyValues = (selectedOptions, relatedOptionsList, stateVariableName) => {
+    if (stateVariableName === 'benefits') {
+      const mappedListItems = selectedOptions.map((option) => {
+        const { id: nameId, defaultMessage: nameDefaultMsg } = relatedOptionsList[option].name.props;
```
Comment (CalebPena) [inline src/Components/Confirmation/Confirmation.js:570]: You can use `relatedOptionsList[option].name` as a formatted message.

File: src/Components/Confirmation/Confirmation.js [hunk 3]
```diff
@@ -564,17 +564,33 @@ const Confirmation = () => {
...
+      });
 
-    return mappedListItems;
+      return mappedListItems;
+    } else {
+      const mappedListItems = selectedOptions.map((option) => {
+        return (
+          <p key={option} className="bottom-margin">
+            {' '}
+            {relatedOptionsList[option]}{' '}
```
Comment (CalebPena) [inline src/Components/Confirmation/Confirmation.js:587]: What do these spaces do?

**Review by CalebPena — CHANGES_REQUESTED** [review]

File: src/Components/Confirmation/Confirmation.js (hunk 3 above)
Comment (ivonne-hernandez) [reply src/Components/Confirmation/Confirmation.js]: They're extraneous so I removed them.

File: src/Components/Confirmation/Confirmation.js (hunk 2 above)
Comment (ivonne-hernandez) [reply src/Components/Confirmation/Confirmation.js]: I made this change.

**Review by CalebPena — APPROVED** [review]

**Pull request merged**